MaterialLib/MPL/CreateProperty.cpp

```diff
@@ -179,7 +186,7 @@ std::unique_ptr<MaterialPropertyLib::Property> createProperty(
             //! \ogs_file_param{properties__property__Parameter__parameter_name}
             config.getConfigParameter<std::string>("parameter_name");
         auto const& parameter = ParameterLib::findParameter<double>(
-            parameter_name, parameters, 1, nullptr);
+            parameter_name, parameters, 0, nullptr);
```

Comment:
What is the reason of this change?

Reply:
The parameter found in the subtree `<parameters>/<parameter>` can be read anyway, irrespective of number of components.

Reply:
@renchao-lu That is true. So the change isn't necessary?

Reply:
@TomFischer 0 means, that the number of components is arbitrary.